docs/blog/2021-10-18-announcing-lunasec.md

```diff
@@ -102,15 +104,15 @@ by Software Developers that aren't experts in security.
 ### We've seen how painful many security tools are to use or migrate to.
 It's easier when you tackle a problem one step at a time. 
 
-That's why LunaSec is [modular](https://www.lunasec.io/docs/pages/overview/security/levels/) so that adoption can be gradual.
+That's why LunaSec is [modular](https://www.lunasec.io/docs/pages/overview/security/levels/), so that its adoption can be gradual.
 
 ### We've seen every company re-invent the wheel time after time.
 That's why LunaSec was made [Open Source](https://github.com/lunasec-io/lunasec/blob/master/LICENSE) under a 
 permissive Open Source license (Apache 2.0).
 
 We hope to build a community of like-minded individuals to make security tooling available for everybody to use.
 
-### We've seen how technical debt bogs down developers and prevents them from fixing bugs (even when they would like to).
+### We've seen how technical departments bog down developers and prevents them from fixing bugs (even when they would like to).
```

Comment:
"tech debt" means something to developers, it was correct before

Reply:
gotcha.  i'll revert.  i assumed it was truncated.